cookbook/docs/weather_forecasting.rst

```diff
@@ -1,8 +1,12 @@
 ###################
-Weather forecasting
+Weather Forecasting
 ###################
 
-Learn how to build an online weather forecasting system that updates a model daily and displays hourly forecasts on a web UI, using Pandera and Streamlit.
+Learn how to build an online weather forecasting system that updates a model daily and displays hourly forecasts on a web UI, using Pandera and Streamlit. 
+
+The video below breifly shows how the Weather Forecasting app is made, a few launch plans, and a Streamlit demo. Find the complete video, `Case Study: Building a Weather Forecasting App Using Flyte and Streamlit <https://youtu.be/c-X1u42uK-g>`_ on YouTube.
```

Comment:
```suggestion
The video below briefly shows how the Weather Forecasting app is made, and a Streamlit demo. Find the complete video, `Case Study: Building a Weather Forecasting App Using Flyte and Streamlit <https://youtu.be/c-X1u42uK-g>`_ on YouTube.
```

Reply:
I don't think that needs to be specified, but up to you.

Reply:
Ok thanks.